src/main/java/com/graphhopper/converter/resources/ConverterResourceNominatim.java

```diff
@@ -0,0 +1,79 @@
+package com.graphhopper.converter.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import com.graphhopper.converter.api.GHResponse;
+import com.graphhopper.converter.api.NominatimEntry;
+import com.graphhopper.converter.api.OpenCageDataResponse;
+import com.graphhopper.converter.api.Status;
+import com.graphhopper.converter.core.Converter;
+import java.util.HashMap;
+
+import javax.validation.constraints.NotNull;
+import javax.ws.rs.*;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ * @author Robin Boldt
+ */
+@Path("/nominatim")
+@Produces(MediaType.APPLICATION_JSON)
+public class ConverterResourceNominatim {
```

Comment:
I need an explicit way to request results from specific geocoders and handle 'defaulting' in the layer above this converter, so I would not need this feature :)

But we can rename to `search` and use the provider argument instead and merge the two resources this way if you like it better.